src/TraceEvent/Computers/TraceProcess.cs

```diff
@@ -27,24 +27,27 @@ namespace Microsoft.Diagnostics.Tracing.Analysis
     /// </summary>
     public static class TraceProcessesExtensions
     {
+        internal static readonly string ProcessesKey = "Computers/Processes";
+
         public static void NeedProcesses(this TraceEventDispatcher source)
         {
             TraceProcesses processes = source.Processes();
-            if (processes == null || m_currentSource != source)
+            if (processes == null)
             {
```

Comment:
Help me understand why the check for the source was removed?  This was explicitly added to avoid adding callbacks more than once on the same source.